client.go

```diff
@@ -120,9 +132,19 @@ func readExpr() <-chan expr {
 		s := bufio.NewScanner(c)
 		for s.Scan() {
 			log.Printf("recv: %s", s.Text())
```

Comment:
TBH I'm not 100% certain on using `recv` actually, the only reason I did was because @gokcehan originally suggested it in https://github.com/gokcehan/lf/pull/1310#issuecomment-1621975494 (open the diff), and I just went with it. I guess it's nice in that it sounds opposite to the `send` command, but at the same time it can also be unclear. I had also originally used `query` before, but other words like `info` or `show` are fine too.

Please let me know what you prefer, I think it's best to decide on the name now because otherwise it will be difficult to change later on.

Reply:
Actually, I think `query` is great. I'm happy with `info` or `show` as well. @gokcehan, are you OK with any of those or do you prefer `recv`?

Reply:
@joelim-work @ilyagr I'm okay with all suggestions.